src/partials/footer-content.hbs

```diff
@@ -29,11 +29,10 @@
         <div class="footer-nav">
           <h4>Interact</h4>
           <ul id="menu-interact" class="menu">
-            <li class="menu-irc-channel"><a href="https://webchat.freenode.net/?channels=owncloud">IRC Channel</a></li>
```

Comment:
What happened to the IRC channel? Is it deprecated?

Reply:
As far as I know, yes. Is it not?

Reply:
According to @lefherz and @michaelstingl the channel was the subject of intense spam and is little used nowadays.